v2/emailpassword/common-customizations/signup-form/adding-fields.mdx

```diff
@@ -2,26 +2,216 @@
 id: adding-fields
 title: Adding Extra Fields
 hide_title: true
+show_ui_switcher: true
 ---
 
-import FrontendSDKTabs from "/src/components/tabs/FrontendSDKTabs"
+import {PreBuiltOrCustomUISwitcher, PreBuiltUIContent, CustomUIContent} from "/src/components/preBuiltOrCustomUISwitcher"
+import FrontendPreBuiltUITabs from "/src/components/tabs/FrontendPreBuiltUITabs";
 import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs"
+import FrontendCustomUITabs from "/src/components/tabs/FrontendCustomUITabs"
+import NpmOrScriptTabs from "/src/components/tabs/NpmOrScriptTabs"
+import AppInfoForm from "/src/components/appInfoForm"
 import TabItem from '@theme/TabItem';
 
 # Adding Extra Fields
 
+<PreBuiltOrCustomUISwitcher>
+
 ## Step 1: Front End
 
+<!-- COPY SECTION -->
+<!-- ./emailpassword/common-customizations/signup-form/adding-fields.mdx -->
+<!-- 1 -->
+
+<CustomUIContent>
+
+<FrontendCustomUITabs>
+<TabItem value="web">
+
+Esnure that your Custom UI calls the following function with additional fields when the user clicks on the sign up button.
+
+<NpmOrScriptTabs>
+<TabItem value="npm">
+
+```tsx
+import { ^{webjsEmailPasswordSignUp} } from "supertokens-web-js/recipe/^{codeImportRecipeName}";
+
+async function signUpClicked(formData) {
+    try {
+        let response = await ^{webjsEmailPasswordSignUp}({
+            formFields: [{
+                id: "email",
+                value: formData.email
+            }, {
+                id: "password",
+                value: formData.password
+            },{
+                id: "name",
+                value: formData.name
+            }, {
+                id: "age",
+                value: formData.age
+            }, {
+                id: "country",
+                value: formData.country
+            }]
+        })
```

Comment:
just this is enough... don't need to show how to handle the response as thats already done in the setup section.